examples/jms-consumer-acknowledgement/jms_consumer_acknowledgement.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on consuming messages from a JMS provider using Ballerina.
```

Comment:
```suggestion
description: BBE on consuming messages from a JMS provider using Ballerina.
```
We need not start every BBE description by saying `BBE on ` as it will be redundant. Check the other BBEs and preserve consistency.

Reply:
Since BBEs related to other messaging connector modules follow the same structure. Will follow the same here as well.